### DISCUSSION TITLE
Comments Area

### DISCUSSION THREAD

**Discussion by rickvanprim** [hugo-sid/hugo-blog-awesome#32] [Feature requests]
It would be nice if this theme had builtin support for a comments section, like the Disqus builtin.

My poor current attempt is appending this to the bottom of `single.html` right before `</article>`:
```html
<footer>
    <hr style="margin-top: 40px; margin-bottom: 40px;"/>
    {{ template "_internal/disqus.html" . }}
</footer>
```
I'm sure someone who's actually good at web styling could do a vastly better job 😅

**Comment by hugo-sid** [comment]
Hi @rickvanprim .

Thanks for your suggestion.

> appending this to the bottom of `single.html` right before `</article>`

You almost did it right (including `{{ template "_internal/disqus.html" . }}`). 

However, I don't think we should be adding another set of `<footer>` tags. 
One pair of footer tag already exists in `layouts/partials/footer.html`.
I think we should modify with this partial.

**Reply by rickvanprim** [reply]
Hey Sid,

I just wrapped it in a `footer` based on the first theme I looked at with it (https://github.com/surajmandalcell/potato-dark/blob/master/layouts/_default/single.html), which had it as a `footer` inside of `article`.  It does seem like it shouldn't be part of `article` though.  Do you think it would make more sense right below `article` without the `footer` wrapping?

Secondly, would you like me to put up a PR for this?  Presumably the styling should get stuck in a `.scss` file somewhere instead of just inline?

Thanks,
James

**Reply by hugo-sid** [reply]
Hi James. Thanks for your message.

> Do you think it would make more sense right below article without the footer wrapping?

Yes, I think so.

> Secondly, would you like me to put up a PR for this? 

You are most welcome to submit a PR for this. We can probably collaborate & fix (if required) in the PR.

Thanks,
Sid

**Reply by rickvanprim** [reply]
PR created https://github.com/hugo-sid/hugo-blog-awesome/pull/44 🙂

**Reply by hugo-sid** [reply]
Thanks for the PR, @rickvanprim.

I will have a look at it.
